index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
+        </a>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </nav>
+    </header>
+    <section id="greeting">
+        <p>I'm John Glad to see you</p>
+    </section>
+    <section id="about">
+        <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </section>
+    <section id="socials">
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div id="social-icons"> 
+            <img src="/media/social media icons.png">
+        </div>
+    </section>
+    <!--<!DOCTYPE html>
```

Comment:
- [ ] As suggested by the previous reviewer, please remove commented out code

Reply:
Hello can you check again @topeogunleye It appears the push did not go through when I sent it. Sorry for the inconvinence caused.

Reply:
@tjay1760, I'll check and get back to you

Reply:
@tjay1760, The change is there now, the commented out code is gone 👏